### DISCUSSION TITLE
How to load CDN only when custom field is used?

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#12614] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.73  

### How can we help you?

My custom field requires inclusion of a CDN script and I have extracted it into an package with its own Service Provider.
How do I get filament to include/load the CDN only when the field is used?

**Comment by jechazelle** [comment, marked as answer]
Did you try it:

custom-componeent.blade.php
```
<div>
    ...
</div>

@assets
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tributejs/3.3.2/tribute.min.css" />
    <script src="https://cdnjs.cloudflare.com/ajax/libs/tributejs/3.3.2/tribute.min.js"></script>
@endassets
```

**Reply by bilogic** [reply]
Oh, thanks but i used the `Js::make()` facade. would this load the CDN with every component instance?

**Reply by bilogic** [reply]
great it works the way I wanted!
